proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -252,17 +295,17 @@ would encode to:
 
 This proposal specifically does not support auto-synthesis for enums with overloaded case identifiers. This decision has been made because there is no clear way to support the feature, while also allowing the model to evolve, without severe restrictions. The separate cases in an enum typically have different semantics associated with them, so it is crucial to be able to properly identify the different cases and reject unknown cases.
 
+#### Evolution with overloaded case identifiers
+
 In this proposal, we are using keys as descriminators. For overloaded case names that would not be sufficient to identify the different overloads. An alternative would be to use the full name, including labels, e.g. `"store(key:,value:)"` for `case store(key: String, value: Int)`. This leads to several problems.
 
 1. Not a valid enum case identifier, so no user customization possible
 2. Not forward/backward compatible because it changes when parameters are added
 3. Very Swift specific, making interop with non-Swift systems awkward
```

Comment:
I'm not sure I understand the third point - the key is a string value.

Reply:
I can remove it. My point was `"store(key:,value:)"` is not something commonly used in other languages and would be a bit weird if it appeared in a serialized document.